### PR TITLE
change lyrics

### PR DESCRIPTION
**Pull request by agoytiarios**
Review my changes, please!

### CODE REVIEW

File: lyrics/metal/nothing-else-matters.txt [hunk 1]
```diff
@@ -27,13 +27,13 @@ And nothing else matters
 Trust I seek and I find in you
 Every day for us something new
 Open mind for a different view
-And nothing else matters
+Everything else matters
```
Comment (PabloUribeB) [inline lyrics/metal/nothing-else-matters.txt]: Not true!! Nothing else matters

File: lyrics/metal/nothing-else-matters.txt [hunk 2]
```diff
@@ -27,13 +27,13 @@ And nothing else matters
...
 Never cared for what they say
 Never cared for games they play
 Never cared for what they do
 Never cared for what they know
 And I know, yeah, yeah
-So close, no matter how far
+So far, no matter how far
```
Comment (PabloUribeB) [inline lyrics/metal/nothing-else-matters.txt]:
```suggestion
So far, no matter how close
```

**Review by PabloUribeB — COMMENTED** [review]
Great!

File: lyrics/metal/nothing-else-matters.txt (hunk 1 above)
Comment (PabloUribeB) [reply lyrics/metal/nothing-else-matters.txt]: I'm confident on this

Comment (Realadeniyi247) [reply lyrics/metal/nothing-else-matters.txt]: great!

Comment (Realadeniyi247) [inline lyrics/metal/nothing-else-matters.txt]:
```suggestion
Everything else matter

```

**Review by Realadeniyi247 — CHANGES_REQUESTED** [review]

**Pull request merged**